examples/dynamic/Makefile

```diff
@@ -0,0 +1,5 @@
+
+public/main.js: views/*
+	mkdir -p tmp
+	./node_modules/.bin/jsx -x jsx views tmp
+	./node_modules/.bin/browserify tmp/main.js -o public/main.js
```

Comment:
Could just combine these into a single step with the `reactify` transform and then you don't need any tmp dirs and could probably drop the Makefile entirely, just putting it all in `npm start`.

You could even just export the function from your `main` module and expose that with browserify instead of setting `window.main` directly.

Further while we're here, it might be interesting to actually do this in express instead of a build step. I think we can get live changes on reload that way. https://github.com/ForbesLindesay/browserify-middleware

Not that this is bad, it's definitely the simplest way to get running but I think jumping to the next step might answer some questions for people (and also, Windows + Makefile don't really get along).